### DISCUSSION TITLE
SVG screenshot option?

### DISCUSSION THREAD

**Discussion by bryanjamesmiller** [spatie/browsershot#655] [Ideas]
Hi, I have a request to take a screenshot and save the image as an SVG.  Is that supported or might it be possible to do so?  Thanks for all your awesome packages!

**Comment by mutschler** [comment]
I don't think i understand why one would do that, since the perks of an SVG is that it's a Vector based format...

You could however make a screenshot and put the generated screenshot in an svg image tag:

```
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <image href="SCREENSHOT.jpg" x="0" y="0" width="IMGWIDTH" height="IMGHEIGHT" />
</svg>
```

you'll need the svg and the image to actually be able to view anything... that could be avoided by passing the image as base64 image:

```
<image href="data:image/jpeg;charset=utf-8;base64,<DATA>" x="0" y="0" width="IMGWIDTH" height="IMGHEIGHT" />
```
